packages/cli-hermes/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@react-native-community/cli-hermes",
+  "version": "4.11.0",
+  "license": "MIT",
+  "main": "build/index.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "types": "build/index.d.ts",
+  "dependencies": {
+    "@react-native-community/cli-tools": "^4.11.0",
+    "hermes-profile-transformer": "^0.0.6",
```

Comment:
missing `chalk` in deps

Reply:
Oh so since `chalk` is a dependency in the general repo, do we need to add it here too?

Reply:
Yes. You will be able to install this package alone, so all dependencies used need to be explicitly declared, because there may not be "general repo" or other packages containing this package.